stanford-earth/SE3_BLT
# EARTH-182: Revise home page setting so it doesn't redirect to canonical url.

# READY FOR REVIEW

# Summary
- Stops the home page from redirecting to `/home`

# Needed By (Date)
- Before July release

# Urgency
- Normal

# Steps to Test

1. Checkout this branch
2. Run `drush csim local -y`
3. Navigate to the root url of your website
4. See that the url stays at the TLD and doesn't redirect to `/home`

# Associated Issues and/or People
- EARTH-182

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
